content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md

```diff
@@ -407,9 +413,12 @@ Lors de l’ installation et de l’ exploitation de ce dispositif, la distance
 
 Hereby, Arduino S.r.l. declares that this product is in compliance with essential requirements and other relevant provisions of Directive 201453/EU. This product is allowed to be used in all EU member states.
 
-| Frequency bands      | Typical Output Power |
-|----------------------|----------------------|
-| 2.4 GHz, 40 channels | TBC                  |
+| Frequency bands                 | Maximum Output Power |
+| ------------------------------- | -------------------- |
+| 2402 MHz ~ 2480 MHz (EDR)       | -0.21 dBM            |
+| 2402 MHz ~ 2480 MHz (BLE)       | 4.79 dBM             |
+| 2412 MHz ~ 2462 MHz (2.4G WiFi) | 16.21 dBM            |
```

Comment:
```suggestion
| 2412 MHz ~ 2462 MHz (2.4GHz WiFi) | 16.21 dBm            |
```

Reply:
Based on https://en.wikipedia.org/wiki/DBm this is correct, added comment about the arduino style guide too